lib/percy/capybara/loaders/filesystem_loader.rb

```diff
@@ -0,0 +1,50 @@
+require 'percy/capybara/loaders/base_loader'
+require 'digest'
+require 'find'
+
+module Percy
+  module Capybara
+    module Loaders
+
+      # Resource loader that looks for resources in the specified folder.
+      class FilesystemLoader < BaseLoader
+        SKIP_RESOURCE_EXTENSIONS = [
+          '.map',  # Ignore source maps.
+          '.gz',  # Ignore gzipped files.
+        ]
+        MAX_FILESIZE_BYTES = 15 * 1024**2  # 15 MB.
+
+        def initialize(options = {})
+          # @asset_path should point to a _compiled_ static assets directory, not source assets.
+          @asset_path = options[:asset_path]
+          super
+        end
+
+        def snapshot_resources
+          [root_html_resource]
+        end
+
+        def build_resources
+          resources = []
+          Find.find(@asset_path).each do |path|
```

Comment:
We may need to absolut-ify this path, not sure if this is going to work for all different path types. Like, if someone does `assets_path: './dist/'` where is it going to be relative to? We may just want to require that people must give an absolute path and handle that part themselves.